docs/installation/kubernetes.mdx

```diff
@@ -0,0 +1,173 @@
+:::
+</TabItem>
+<TabItem value="docker">
+
+If you have docker environment, you can easily run CLI with the vela CLI docker image called `oamdev/vela-cli`:
+
+```
+$ docker run --rm -it -v ~/.kube:/root/.kube oamdev/vela-cli version
+CLI Version: master
+Core Version: v1.3.4
+GitRevision: git-1d823780
+GolangVersion: go1.17.10
+```
+
+Refer to [using Vela CLI docker image](../platform-engineers/system-operation/vela-cli-image) for more usage.
+
+</TabItem>
+</Tabs>
+
+### 3. Install KubeVela Core
+
+<Tabs
+    className="unique-tabs"
+    defaultValue="vela"
+    values={[
+      {label: 'Default', value: 'vela'},
+      {label: 'Helm', value: 'helm'},
+    ]}>
+<TabItem value="vela">
+
+```shell script
+$ vela install
+```
+<details>
+  <summary> check out the outcome </summary>
+
+  ```shell
+  ...
+  Watching for changes to Job kubevela-vela-core-cluster-gateway-tls-secret-patch with timeout of 18m0s
+  Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: ADDED
+  kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 0, jobs failed: 0, jobs succeeded: 0
+  Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
+  kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 1, jobs failed: 0, jobs succeeded: 0
+  Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
+  kubevela-vela-core-cluster-gateway-tls-secret-patch: Jobs active: 1, jobs failed: 0, jobs succeeded: 0
+  Add/Modify event for kubevela-vela-core-cluster-gateway-tls-secret-patch: MODIFIED
+  Starting delete for "kubevela-vela-core-admission" ServiceAccount
+  Starting delete for "kubevela-vela-core-admission" ClusterRole
+  Starting delete for "kubevela-vela-core-admission" ClusterRoleBinding
+  Starting delete for "kubevela-vela-core-admission" Role
+  Starting delete for "kubevela-vela-core-admission" RoleBinding
+  Starting delete for "kubevela-vela-core-admission-patch" Job
+  Starting delete for "kubevela-vela-core-cluster-gateway-admission" ServiceAccount
+  Starting delete for "kubevela-vela-core-cluster-gateway-admission" Role
+  Starting delete for "kubevela-vela-core-cluster-gateway-admission" RoleBinding
+  Starting delete for "kubevela-vela-core-cluster-gateway-tls-secret-patch" Job
+
+  KubeVela control plane has been successfully set up on your cluster.
+  If you want to enable dashboard, please run "vela addon enable velaux"
+  ```
+
+  </details>
+
+</TabItem>
+<TabItem value="helm">
+
+If you are helm user, you can also use helm to install KubeVela core:
+
+:::note
+helm v3.2.0+ required
+:::
+
+```
+helm repo add kubevela https://charts.kubevela.net/core
+helm repo update
+helm install --create-namespace -n vela-system kubevela kubevela/vela-core --wait
+```
+
+</TabItem>
+</Tabs>
+
+### 4. Install VelaUX
+
+Please refer to [VelaUX Guide](../reference/addons/velaux).
+
+### 5. Advanced Installation
```

Comment:
Why do we need this step?

Reply:
Maybe replace the title: "Upgrade. uninstallation and other installation options"?